assignment_9/info.txt

```diff
@@ -0,0 +1,6 @@
+5.006     3.428      1.462     0.246
+0.352     0.379      0.174     0.105
+5.936     2.770      4.260     1.326
+0.516     0.314      0.470     0.198
+6.588     2.974      5.552     2.026
+0.636     0.322      0.552     0.275
```

Comment:
Is this tab separated or space separated?

Reply:
Space separated values.